jmetal-core/src/main/java/org/uma/jmetal/util/experiment/component/EvolutionChart/HistoricAlgorithm.java

```diff
@@ -0,0 +1,41 @@
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package org.uma.jmetal.util.experiment.component.EvolutionChart;
+
+import java.util.List;
+import java.util.Map;
+import org.uma.jmetal.solution.DoubleSolution;
+
+/**
+ *
+ * @author Lucas Prestes <lucas.prestes.lp@gmail.com> 
+ */
+public interface HistoricAlgorithm {
```

Comment:
I don't get the purpose of having an algorithm interface to make an experiment tool available. We must ensure that dependencies restrict to the minimum, so we cannot impose to an algorithm to implement a specific interface for the sole purpose of using one experiment component (this is what `HistoricAlgorithm` makes me suppose with its naming). Rather, the component should be independent, and it should be able to receive data whatever the source, so we can choose for instance to feed it directly with the outputs of a running algorithm (or with its measures) or feed it with already stored data. Can you imagine if we implement an interface for each component we want to experiment? Basically, a good experiment component should be algorithm-agnostic.